bgpd/bgp_zebra.c

```diff
@@ -3732,38 +3732,26 @@ void bgp_zebra_announce_default(struct bgp *bgp, struct nexthop *nh,
 	api.distance = ZEBRA_EBGP_DISTANCE_DEFAULT;
 	SET_FLAG(api.message, ZAPI_MESSAGE_DISTANCE);
 
+	api_nh->vrf_id = nh->vrf_id;
+
+	if (BGP_DEBUG(zebra, ZEBRA)) {
+		struct vrf *vrf;
+
+		vrf = vrf_lookup_by_id(nh->vrf_id);
+		zlog_debug("%s: %s default route to %pNHvv(%s) table %d",
+			   bgp->name_pretty, announce ? "adding" : "withdrawing",
+			   &nh, VRF_LOGNAME(vrf), table_id);
```

Comment:
Seems double pointer is passed for `nh`?

Reply:
yeah I noticed that in the build failure :(  Not sure why it worked locally

Reply:
fixed